src/App.js

```diff
@@ -1,27 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
+import {
+  NavDisplay, LeftSection, Operators,
+} from './components/components';
 
```

Comment:
- [ ] According to the project requirement, when you will create your Calculator and it is ready, import it and use it in your top-level component: App.js. Please fix it in order to make your app follow the project requirement 👍

Reply:
@codecaiine Thank you for reminding me of those, Meanwhile, I want to ask a question, Should I still render the `App` Component in the `index.js` file